tests/eko/test_matching_n3lo.py

```diff
@@ -173,47 +163,46 @@ def test_Blumlein_3(get_sx):
             # as non singlet. The accuracy is worst for large N
             # due to the approximations of F functions.
             np.testing.assert_allclose(
-                aS3[1, 1], ref_val_qqNS[L][idx] + ref_val_qqPS[L][idx], rtol=3e-2
+                aS3[1, 1], ref_val_qqNS[L][idx] + ref_val_qqPS[L][idx], rtol=8e-2
             )
 
     # Here we test the critical parts
     nf = 3
     ref_ggTF_app = [-28.9075, -180.659, -229.537, -281.337, -467.164]
     for idx, N in enumerate([2.0, 4.0, 6.0, 10.0, 100.0]):
-        sx = get_sx(N)
-        smx = get_smx(N)
-        s3x = get_s3x(N, sx, smx)
-        s4x = get_s4x(N, sx, smx)
-        Aggtf2 = as3.aggTF2.A_ggTF2(N, sx, s3x)
+        sx_cache = compute_harmonics_cache(N, 3, True)
+        Aggtf2 = as3.aggTF2.A_ggTF2(N, sx_cache)
         if N != 100:
             # Limited in the small N region
             np.testing.assert_allclose(Aggtf2, ref_val_ggTF2[0][idx], rtol=15e-2)
         np.testing.assert_allclose(Aggtf2, ref_ggTF_app[idx], rtol=2e-4)
 
         np.testing.assert_allclose(
-            as3.agg.A_gg(N, sx, smx, s3x, s4x, nf, L=0) - Aggtf2,
+            as3.agg.A_gg(N, sx_cache, nf, L=0) - Aggtf2,
             ref_val_gg[0][idx],
             rtol=3e-6,
         )
 
     # odd numbers of qqNS
     # Limited accuracy due to F functions
     ref_qqNS_odd = [-40.94998646588999, -21.598793547423504, 6.966325573931755]
-    for N, ref in zip([3.0, 15.0, 101.0], ref_qqNS_odd):
-        sx = get_sx(N)
-        smx = get_smx(N)
-        s3x = get_s3x(N, sx, smx)
-        s4x = get_s4x(N, sx, smx)
+    rtols = [4e-4, 3e-3, 21e-2]
```

Comment:
The last accuracy is really bad...

Reply:
Indeed there was a dot missing so it's `2.1e-2`, not perfect but better than before...